Gopkg.toml

```diff
@@ -29,7 +29,7 @@ required = ["github.com/3rf/mongo-lint"]
 
 [[constraint]]
   name = "github.com/mongodb/mongo-tools-common"
-  version = "v3.0.7"
+  revision = "2ecb309551185b1ca80cb3c52e2cafe19e0e607c"
```

Comment:
Temporarily pointing to [TOOLS-2369 in MTC](https://github.com/mongodb/mongo-tools-common/pull/36).